sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -390,11 +390,17 @@ - (NSString *) version {
   return OO_SKIN_VERSION;
 }
 
-- (void)queryState {
+- (void)onUIReady {
+  if (_player.currentItem) {
+    // embedcode already set, send current item information
+    NSNotification *notification = [NSNotification notificationWithName:OOOoyalaPlayerCurrentItemChangedNotification object:nil];
+    [self bridgeErrorNotification:notification];
+  }
+
   if (_player.state == OOOoyalaPlayerStateError) {
     NSNotification *notification = [NSNotification notificationWithName:OOOoyalaPlayerErrorNotification object:nil];
     [self bridgeErrorNotification:notification];
-  } else {
+  } else if (_player.state != OOOoyalaPlayerStateReady) {
```

Comment:
In JS, using the logical operator == or != sometimes may return an unexpected value, because it will try to do type conversion if the types are different. (ref: http://www.c-point.com/javascript_tutorial/jsgrpComparison.htm). I think we should try to always use === or !==, because this is safer, and is the equivalent to what we use in other languages.